service/history/hsm/tree.go

```diff
@@ -185,22 +212,52 @@ func (n *Node) Path() []Key {
 
 // Outputs returns all outputs produced by transitions on this tree.
 func (n *Node) Outputs() []PathAndOutputs {
-	var paos []PathAndOutputs
-	if len(n.cache.outputs) > 0 {
-		paos = append(paos, PathAndOutputs{Path: n.Path(), Outputs: n.cache.outputs})
+	root := n.root()
+	if root.opLog == nil {
```

Comment:
This should never happen, right? Seems like it'd be better to assume this is never nil.

Reply:
Fair enough

Reply:
Done